src/mmw/js/src/main_water_balance.js

```diff
@@ -39,10 +39,10 @@ var initialize = function(model) {
 
         var result = model[soil][land][precip];
 
-        $precipText.text(precip + '"');
-        $evapoText.text(result.et + '"');
-        $runoffText.text(result.r + '"');
-        $infilText.text(result.i + '"');
+        $precipText.text(convertToMetric(precip) + 'cm');
```

Comment:
There should always be a space between the numeric value and the unit. See http://physics.nist.gov/cuu/Units/checklist.html#15 "Unit Spacing":

> There is a space between the numerical value and unit symbol, even when the value is used in an adjectival sense, except in the case of superscript units for plane angle.
> 
> proper: 	a 25 kg sphere
> an angle of 2° 3'  4"
> If the spelled-out name of a unit is used, the normal rules of English apply: "a roll of 35-millimeter film."